src/Clipboard.js

```diff
@@ -0,0 +1,9 @@
+
+
+export function supportsClipboard() {
```

Comment:
This needs a better name... all the browsers support clipboard to some extent, we're checking specifically for `ClipboardItem`.